heartbeat/monitors/active/http/http_test.go

```diff
@@ -163,7 +163,7 @@ func respondingHTTPBodyChecks(body string) validator.Validator {
 func respondingHTTPHeaderChecks() validator.Validator {
 	return lookslike.MustCompile(map[string]interface{}{
 		"http.response.headers": map[string]interface{}{
-			"Date":           isdef.IsString,
+			"Date":           isdef.Optional(isdef.IsString),
```

Comment:
Porting this change from main: https://github.com/elastic/beats/blob/5e71fe7a7947b844013218a8fe02dce04a142bd9/heartbeat/monitors/active/http/http_test.go#L168